lib/DDG/Spice/Edrpou.pm

```diff
@@ -0,0 +1,27 @@
+package DDG::Spice::Edrpou;
+
+# ABSTRACT: Get information about Ukrainian companies by EDRPOU code
+
+use DDG::Spice;
+use strict;
+use warnings;
+
+# Caching - http://docs.duckduckhack.com/backend-reference/api-reference.html#caching
+spice is_cached => 1;
+spice proxy_cache_valid => '200 10d'; # defaults to this automatically
+
+spice wrap_jsonp_callback => 1;
+
+spice to => 'https://edr.data-gov-ua.org/api/companies?where={%22edrpou%22:%22$1%22}';
+
+# Triggers - https://duck.co/duckduckhack/spice_triggers
+triggers startend => "EDRPOU","ЄДРПОУ","ЄДР","edrpou", "єдрпоу", "єдр";
+
+# Handle statement
+handle remainder => sub {
+    # Trigger if query includes an 8 digit number
+    return $_ if $_ =~ /\d{8}/;
```

Comment:
hmm... I may have been partially wrong on my interpretation of this.

I'm noticing that the above won't trigger with a 7 digit number but it will trigger with a 9 digit number. To match exact try this:

    return $_ if $_ =~ /\d{8}/;

To match only between 5 and 8:

    return $_ if $_ =~ /\d{5,8}/;